shell/platform/glfw/flutter_glfw_private.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_GLFW_FLUTTER_GLFW_PRIVATE_H_
+#define FLUTTER_SHELL_PLATFORM_GLFW_FLUTTER_GLFW_PRIVATE_H_
+
+#include <list>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "flutter/shell/platform/embedder/embedder.h"
+
+const char* GetLocaleStringFromEnvironmentVariables(const char* language,
+                                                    const char* lc_all,
+                                                    const char* lc_messages,
+                                                    const char* lang);
+
+void ParseLocale(const std::string& locale,
+                 std::string* language,
+                 std::string* territory,
+                 std::string* codeset,
+                 std::string* modifier);
+
+std::vector<std::unique_ptr<FlutterLocale>> GetLocales(
+    const char* locale_string,
+    std::list<std::string>& locale_storage);
+
```

Comment:
The reason there are multiple exposed helpers on Windows is that we can't easily mock out what the OS returns; that's not the case here, since you can easily set environment variables in tests, so I would expect there to only be one visible function for getting locales even if it has helper functions internally.

Reply:
I was trying to not set environment variables in tests and instead make more pure functions for testability, but that is more noise than its worth here so I will collapse things.